services/news/src/views/uc-publisher-ads-req.ejs

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title><%= title %></title>
+  <link rel="stylesheet" href="/css/global.css" />
+  <link rel="stylesheet" href="/css/hide-button-style.css">
+  <link rel="icon" href="/img/spy.svg" />
+  <meta http-equiv="origin-trial" content="<%= NEWS_TOKEN %>">
+  <script src="<%= `https://${DSP_HOST}:${EXTERNAL_PORT}/js/dsp.js` %>" dsp="<%= `${DSP_HOST}:${EXTERNAL_PORT}` %>" bucket="<%= `${bucket}` %>" cloudenv="<%= `${cloudEnv}` %>"></script>
+</head>
+
+<body class="container mx-auto flex flex-col gap-6 font-serif sm:w-full md:w-full lg:w-4/5 bg-slate-50 pt-8">
+  <header class="flex flex-col items-center gap-4">
+    <h1 class="text-6xl">Daily Privacy News</h1>
+    <h2 class="text-2xl">A news demo site which allows blocking of ads</h2>
+    <hr class="w-full">
+  </header>
+  <main class="flex flex-col lg:flex-row justify-between gap-6">
+    <article class="flex flex-col gap-6 text-xl leading-6 w-full lg:w-4/6">
+      <p><%= lorem %></p>
+      <div class="hide-button-container">
+        <a href="/uc-publisher-ads-req" class="hide-button">Show all shoes</a>
+        <a href="/uc-publisher-ads-req?excludeProductTag=red_shoe" class="hide-button">Hide red shoes</a>
+        <a href="/uc-publisher-ads-req?excludeProductTag=blue_shoe" class="hide-button">Hide blue shoes</a>
+        <a href="/uc-publisher-ads-req?excludeProductTag=brown_shoe" class="hide-button">Hide brown shoes</a>
+        <a href="/uc-publisher-ads-req?excludeProductTag=sports_shoe" class="hide-button">Hide sports shoes</a>
```

Comment:
NTH: can we do something like combination of the tags?
e.g. red_shoe & sports_shoe

Reply:
We could, but I feel like the extra complexity isn't needed imho.